src/Compute/Compute/VirtualMachine/Operation/NewAzureVMCommand.cs

```diff
@@ -1456,6 +1458,23 @@ private string GenerateOrFindSshKey()
                     }
                     Console.WriteLine("Private key is saved to " + privateKeyFilePath);
 
+                    FileSecurity fileSecurity = new FileSecurity(privateKeyFilePath, AccessControlSections.Access);
```

Comment:
Are these related to your PR??

Reply:
yes, I'm merging changes for SSH key into this PR to avoid merge issues.